website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -29,21 +29,21 @@ metadata:
   name: default
```

Comment:
These changes look good to me, but looks like you have a bunch of whitespace changes in files where you actually didn't make any edits.

Reply:
Are you referring to the two docs files? Both of these have edits, it's just a single edit to remove the `--container-runtime` flag from the default user data section.

Reply:
Got it. I just did a cursory look over it so I missed that change due to all of the whitespace updates. Not sure if there is a way to reduce the number of whitespace updates that are propagated from your editor. For now, I just enabled the "Hide Whitespace" setting and was able to see it more clearly.